### PR TITLE
Finished Skill tree/list

### PR DESCRIPTION
**Pull request by uchennaponuigbo**
Player will start with 10 skill points. Upon each level that is even, the player gains 10 more skill points. At max level, player receives 1 skill point. This totals to 51 skill points to spend, which is enough to max all abilities.

Press "N" on the keyboard to open the menu (which also pauses the game). Navigate the menu with WASD or arrow keys, and press Enter on a skill you want to learn.

### CODE REVIEW

**Comment by Ghop11** [comment]
Do not merge unless it has been resolved and tested locally. Make sure things are not broken.

**Comment by Ghop11** [comment]
let me review it before merging

File: the-lost-penguin/Assets/Scripts/PlayerStats.cs
Comment (Ghop11) [inline the-lost-penguin/Assets/Scripts/PlayerStats.cs:110]: Has the been addressed?

Comment (uchennaponuigbo) [reply the-lost-penguin/Assets/Scripts/PlayerStats.cs:110]: moved to SkillUI.cs

File: the-lost-penguin/Assets/Scripts/SkillUI.cs
Comment (Ghop11) [inline the-lost-penguin/Assets/Scripts/SkillUI.cs]: Needs to be changed to 0.2 or 0.3

**Review by Ghop11 — CHANGES_REQUESTED** [review]

**Review by Ghop11 — APPROVED** [review]
looks good.

**Pull request merged**
